.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.2
```

Comment:
apparently this file is used by `rbenv` and `rvm` to work out the correct ruby version to use when running locally